File: semester_1/python_programming/CYCLE 3/10.py
n = int(input("Enter a number : "))
print("Factors of ", n ," = ", end = " ")

#for i in range(1, n + 1):
#       if n % i == 0:
#               print(i, end = " ")
#using while loop
count = 1
while count <= n:
        if n % count == 0:
                print(count, end = " ")
        count = count + 1

File: semester_1/web_technology/PHP PRACTICE/10.php
<html>
<head>
    <title>Sum Primes</title>
</head>
<body>
    <form method="post">
        <label for="number">Enter a number (N):</label>
        <input type="number" name="number" required>
        <button type="submit">Calculate Sum</button>
    </form>

    <?php
        if ($_SERVER["REQUEST_METHOD"] == "POST") {
            $N = $_POST["number"];
            function isPrime($num) {
                if ($num <= 1) return false;
                for ($i = 2; $i <= sqrt($num); $i++) {
                    if ($num % $i == 0) return false;
                }
                return true;
            }
            $sum = 0;
            echo "prime numbers are : ";
            for ($i = 2; $i <= $N; $i++) {
                if (isPrime($i)) {
                	$sum += $i;
                	echo "$i  ";
                }
            }

            echo "</br> sum = $sum";
        }
    ?>
</body>
</html>